src/ButtonComponent/Button.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { mount, render } from 'enzyme';
+import { block as ButtonComponent, defaultConfig } from './index';
+import { ButtonProps } from './types';
+import { mockGlobalSettings as globalSettings } from './__fixtures__/mockGlobalSettings';
+
+describe('Button component', () => {
+    let props: ButtonProps;
+
+    beforeEach(() => {
+        props = {
+            ...defaultConfig,
+            globalSettings
+        };
+    });
+
+    it('matches snapshot', () => {
+        const rendered = render(<ButtonComponent {...props} />);
+        expect(rendered).toMatchSnapshot();
+    });
+
+    it('uses LinkButton when "href" is passed', () => {
+        const href = '#';
+        const rendered = mount(<ButtonComponent {...props} href={href} />);
+        expect(rendered.find('a').exists()).toBe(true);
```

Comment:
Let's add another snapshot here, to show the HTML structure of the Link Button.

Reply:
@cameron-jones-volusion I used `.toBeTruthy` on my tests that looked for the existance of an element. Do we have a preference for that or the way Jason has it here?